homeassistant/components/launch_library/sensor.py

```diff
@@ -72,25 +76,55 @@ async def async_setup_entry(
 
     async_add_entities(
         [
-            NextLaunchSensor(coordinator, entry.entry_id, name),
+            NextLaunchSensor(
+                coordinator,
+                entry.entry_id,
+                SensorEntityDescription(
+                    key="next_launch",
+                    icon="mdi:rocket-launch",
+                    name=name,
+                ),
+            ),
         ]
     )
```

Comment:
Should I format this differently so it looks nicer?

Reply:
Name shouldn't be part of the EntityDescription.
The EntityDescription should be defined as a constant outside of the setup method.

Reply:
Please see my comment below

Reply:
And i will move the `EntityDescription` to a const

Reply:
Ignore my comments, I see what you mean now 😆